### ISSUE TITLE
Does Current Email Trigger supports Oauth2 Authentication

### ISSUE THREAD

**Issue by byteshiva** [actionsflow/actionsflow#26]
## Description

Does the current Email Trigger support OAuth2 Authentication. I see it has only IMAP Support.

**Comment by theowenyoung** [comment]
Yes, only IMAP is supported. I suppose If Actionsflow supports OAuth2, we have to do Oauth2 flow somewhere, and get the token, then Actionsflow can use the token to do something.
